src/commands/webhooks/addwebhook.js

```diff
@@ -89,8 +89,9 @@ module.exports = class AddWebhook extends Command {
 
   async addWebhook(message, board, userData, trello, _) {
     const availableChannels = this.sortChannels(message.channel.guild.channels)
-      .filter(channel => (channel.type === 5 || channel.type === 0) &&
-        channel.permissionsOf(this.client.user.id).has('manageWebhooks'));
+      .filter(async channel => (channel.type === 5 || channel.type === 0) &&
+        channel.permissionsOf(this.client.user.id).has('manageWebhooks') &&
+        (await channel.getWebhooks()).length < 10);
```

Comment:
In a guild with multiple channels will quickly hit ratelimits. Would probably be best to fetch webhooks for the whole guild and filter out webhooks from there.